### PR TITLE
fix bug of subject admin page rewriting subject name as

### PR DESCRIPTION
**Pull request by annyhe**
relatedLinks name
- fix bug of inputs with identical name overriding each other
- fix lint errors in nearby files

### CODE REVIEW

File: view/admin/utils/getFormData.js [hunk 1]
```diff
@@ -154,14 +179,15 @@ function getFormData(form, aspectRangeFormat, propertyMetaData) {
   if (aspectRangeFormat === 'BOOLEAN') {
     fromBooleanInput(inputs, jsonData);
   } else if (aspectRangeFormat === 'NUMERIC') {
-    fromTextInput(inputs, jsonData, function(value) {
+    fromTextInput(inputs, jsonData, (value) => {
```
Comment (iamigo) [inline view/admin/utils/getFormData.js]:
```
fromTextInput(inputs, jsonData, (value) => parseFloat(value));
```

File: view/admin/utils/getFormData.js [hunk 2]
```diff
@@ -154,14 +179,15 @@ function getFormData(form, aspectRangeFormat, propertyMetaData) {
...
       return parseFloat(value);
     });
   } else if (aspectRangeFormat === 'PERCENT') {
-    fromTextInput(inputs, jsonData, function(value) {
-      return value/100;
+    fromTextInput(inputs, jsonData, (value) => {
```
Comment (iamigo) [inline view/admin/utils/getFormData.js]:
```
fromTextInput(inputs, jsonData, (value) => value / 100);
```

File: view/admin/utils/getFormData.js [hunk 3]
```diff
@@ -80,9 +85,19 @@ function fromTextInput(inputs, jsonData, alterDataFunc) {
  * @returns {Object} with keys to particular DOM elements
  */
 function getInputsAndSelects(form) {
```
Comment (iamigo) [inline view/admin/utils/getFormData.js]: Any way to add a test for this fix?

**Comment by annyhe** [comment]
Demo app deployed to http://refocus-pr-249.herokuapp.com/subjects

File: view/admin/utils/getFormData.js (hunk 3 above)
Comment (annyhe) [inline view/admin/utils/getFormData.js]: There are ways but they are verbose to write and maintain. The crux of the method is node.contains, which is well-documented https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

**Pull request merged**